ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -717,11 +713,7 @@ end
 
 module AuthADWinbind : Auth_signature.AUTH_MODULE = struct
   let get_subject_identifier' subject_name =
-    let* domain =
-      try Ok (get_domain_info_from_db ()).service_name with e -> Error e
-    in
-    let subject_name = domainify_uname ~domain subject_name in
-    Wbinfo.sid_of_name subject_name
+    undomainify_uname subject_name |> Ldap.sid_of_name
```

Comment:
What if you have the same username in 2 different domains?
Will this allow `DOMAIN_B\user` to login with the privileges of `DOMAIN_A\user` but with the password of `DOMAIN_B\user`?

Reply:
Here is not writing the `/etc/pam.d/hcp_user`, 
`/etc/pam.d/hcp_users` has domain name prefixed.